README.md

```diff
@@ -35,18 +29,11 @@ reflective of the final deployment environment or AOT deployment. Supported
 runtimes:
 
 * JAX, Tensorflow, PyTorch and TFLite with IREE using MLIR input.
-* JAX, Tensorflow with XLA using HLO input.
```

Comment:
@mariecwhite ditto from above

Reply:
This functionality is also still available.